.pre-commit-config.yaml

```
@@ -8,7 +8,7 @@ default_language_version:
 repos:
 
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.3.0
+  rev: v4.4.0
```

```suggestion
  rev: v4.3.0
```
Reverting this because there seems to be an issue with the new version of `check-toml`.